gems/quilt_rails/lib/generators/quilt/install_generator.rb

```diff
@@ -0,0 +1,42 @@
+module Quilt
+  class QuiltInstallGenerator < Rails::Generators::Base
```

Comment:
I'd like to name this `InstallGenerator`, but when I do all of the tests fail because the files that are written cannot be found:

```
Failure:
InstallGeneratorTest#test_adds_engine_to_routes_if_file_already_exists [/Users/mattseccafien/src/github.com/Shopify/quilt/gems/quilt_rails/test/generators/quilt/install_generator_test.rb:33]:
Expected file "config/routes.rb" to exist, but does not
```

@rafaelfranca do you know what I am missing here?

Reply:
I just tried this diff and everything passes:

```diff
diff --git a/gems/quilt_rails/lib/generators/quilt/install_generator.rb b/gems/quilt_rails/lib/generators/quilt/install_generator.rb
index 2c014678..ae8630a3 100644
--- a/gems/quilt_rails/lib/generators/quilt/install_generator.rb
+++ b/gems/quilt_rails/lib/generators/quilt/install_generator.rb
@@ -1,5 +1,5 @@
 module Quilt
-  class QuiltInstallGenerator < Rails::Generators::Base
+  class InstallGenerator < Rails::Generators::Base
     source_root File.expand_path('templates', __dir__)
 
     desc "This generator mounts the Quilt engine and adds a React app."
diff --git a/gems/quilt_rails/test/generators/quilt/install_generator_test.rb b/gems/quilt_rails/test/generators/quilt/install_generator_test.rb
index 98f36687..2b371729 100644
--- a/gems/quilt_rails/test/generators/quilt/install_generator_test.rb
+++ b/gems/quilt_rails/test/generators/quilt/install_generator_test.rb
@@ -4,7 +4,7 @@ require 'rails/generators'
 require 'generators/quilt/install_generator'
 
 class QuiltInstallGeneratorTest < Rails::Generators::TestCase
-  tests Quilt::QuiltInstallGenerator
+  tests Quilt::InstallGenerator
   destination File.expand_path("../tmp", File.dirname(__FILE__))
 
   setup do
```

Reply:
@rafaelfranca the CI failures reflects what I am seeing locally, https://travis-ci.org/Shopify/quilt/jobs/573957513#L312

Anything I can do to trouble shoot this?

Reply:
It happens because you have two files in different places defining the same test class:

Both gems/quilt_rails/test/generators/quilt/install_generator_test.rb and gems/quilt_rails/test/generators/sewing_kit/install_generator_test.rb defines a `InstallGeneratorTest` and them you have different results depending on which one is loaded fist. The name of classes need to be unique.

Reply:
@rafaelfranca thank you for being so patient here! That was it. 🙇